OmiseSwift/Balance.swift

```diff
@@ -13,9 +13,9 @@ public class Balance: ResourceObject {
         set { set("total", Int64Converter.self, toValue: newValue) }
     }
     
-    public var currency: String? {
-        get { return get("currency", StringConverter.self) }
-        set { set("currency", StringConverter.self, toValue: newValue) }
+    public var currency: Currency? {
+        get { return get("currency", StringConverter.self).flatMap(Currency.init(code:)) }
```

Comment:
@pitiphong-p currency is used in many places, maybe it warrants a `CurrencyConverter.self`

implementation is quite simple.